prowler/lib/mutelist/models.py

```diff
@@ -1,24 +0,0 @@
-from schema import Optional, Schema
-
-mutelist_schema = Schema(
-    {
-        "Accounts": {
-            str: {
-                "Checks": {
-                    str: {
-                        "Regions": list,
-                        "Resources": list,
-                        Optional("Tags"): list,
-                        Optional("Exceptions"): {
-                            Optional("Accounts"): list,
-                            Optional("Regions"): list,
-                            Optional("Resources"): list,
-                            Optional("Tags"): list,
-                        },
-                        Optional("Description"): str,
```

Comment:
Can you verify how these fields can be set as optional in jsonschema?

Reply:
Done in the next commit!